artifacts/definitions/Linux/Events/ExecutableFiles.yaml

```diff
@@ -0,0 +1,49 @@
+name: Linux.Events.ExecutableFiles
+description: |
+  This artifact collects Every new executable file in /tmp or login
+  directory from the Linux kernel.
+
+  This artifact relies on the presence of `auditctl` usually included
+  in the auditd package. On Ubuntu you can install it using:
+
+  ```
+  apt-get install auditd
+  ```
+
+  On SUSE operating systems you can install it using:
+  ```
+  zypper install audit
+  ```
+
+precondition: SELECT OS From info() where OS = 'linux'
+
+type: CLIENT_EVENT
+
+required_permissions:
+  - EXECVE
+
+parameters:
+  - name: rules
+    description: List of rules to be installed
+
+sources:
+  - query: |
+     LET exec_bit_log = SELECT timestamp(string=Timestamp) AS Time, Sequence,
+           atoi(string=Summary.Actor.Primary) AS UserId,
+           Result AS State,
+           File.path AS FileName,
+           join(array=[Process.CWD, File.path], sep='/') AS FilePath,
+           hash(path=join(array=[Process.CWD, File.path], sep='/')) AS Hash
+       FROM audit(rules=["-a always,exit -F arch=b64 -S fchmodat -F a2&73 -k vrr_exec_bit_addition", "-a always,exit -F arch=b32 -S fchmodat -F a2&73 -k vrr_exec_bit_addition", "-a always,exit -F arch=b64 -S fchmod -F a1&73 -k vrr_exec_bit_addition", "-a always,exit -F arch=b32 -S fchmod -F a1&73 -k vrr_exec_bit_addition"])
```

Comment:
Would it make sense to use a LET variable to hold the rules array to clean this up here and elsewhere?